app/pathogen/arbovirus/dashboard/(map)/MapExpandPlotsPrompt.tsx

```diff
@@ -0,0 +1,21 @@
+import { Card, CardContent } from "@/components/ui/card";
+
+interface MapExpandPlotsPromptProps {
+  onClick: () => void;
+  hidden?: boolean;
+  text: string;
+}
+
+export const MapExpandPlotsPrompt = ({
+  onClick,
+  hidden,
+  text,
+}: MapExpandPlotsPromptProps) => {
+  return (
+    <Card className={hidden ? "hidden" : "inline-block"}>
+      <CardContent className={"flex w-fit p-2 hover:bg-black/5 rounded-lg"}>
+        <button onClick={onClick}> <p> {text} </p> </button>
+      </CardContent>
+    </Card>
```

Comment:
I think the font of this text should change: right now, it does not immediately occur to me as a user that this is a clickable  element even just at a glance. Maybe we have have it expand a little when you hover over it to prompt people to click it? I also think we should change the text to either have a separate font, or maybe have a `<-` or `<<` to indicate that you are going to expand a UI element on the left.